CONTRIBUTING.md

```diff
@@ -124,19 +124,6 @@ NB: reviewers may run more complex test/CI scripts, thus, satisfying all the
 requirements above is just a preliminary, but not necessary sufficient step for
 getting the PR accepted as a valid candidate PR for the `master` branch.
 
-PR authors may also find it useful to run the following script locally in order
```

Comment:
guess this isn't useful anymore?

Reply:
No, because it is out of sync with what's actually run in CI (it already does not include the `base58` crate for example)

Reply:
The individual scripts in each crate are out of date but the top one is up to date.